README.md

```diff
@@ -185,7 +199,7 @@ proxy [command options] [arguments...]
 
 * [GOlang](https://golang.org/dl/) - Programming Language
 * [urfave/cli](https://github.com/urfave/cli) - The CLI framework
-* [Travis](https://maven.apache.org/) - Conitnuous Integration Tool
+* [Travis](https://maven.apache.org/) - Continuous Integration Tool
```

Comment:
Is this the correct Travis UI URL?

Reply:
Good pickup, not sure why it's got that url >.<